examples/https-client/https_client.out

```diff
@@ -0,0 +1,6 @@
+# Before testing this sample, first start a sample HTTPS service.
```

Comment:
Shall we couple this with `https_listener.bal`?
By adding a comment like,
As a prerequisite, navigate to the directory that contains the `https_listener.bal` file and use the `bal run` command below. (You may need to change the trusted certificate file path.)
bal run https_listener.bal

WDYT?

Reply:
+1. Will update accordingly.

Reply:
Fixed with https://github.com/ballerina-platform/ballerina-distribution/pull/1708/commits/abf26ebad164a7f14dcf9d491aaf4a647984c4fe